v3/lints/cabf_br/lint_ca_organization_name_missing.go

```diff
@@ -23,16 +23,51 @@ import (
 type caOrganizationNameMissing struct{}
 
 /************************************************
-BRs: 7.1.2.1e
-The Certificate Subject MUST contain the following: organizationName (OID 2.5.4.10): This field MUST be present and the contents MUST contain either the Subject CA’s name or DBA as verified under Section 3.2.2.2.
+--- Citation History of this Requirement ---
+v1.0 to v1.2.4:   9.1.3
+v1.2.5:           Appendix B 1E, 2H
```

Comment:
I am VERY impressed that you have managed to track down not only the latest citation, but what is essentially the migration history as well.